dotCMS/src/main/java/com/dotmarketing/filters/CMSFilter.java

```diff
@@ -70,11 +66,18 @@ public void destroy() {
 
     private static VisitorAPI visitorAPI = APILocator.getVisitorAPI();
 
+    @Override
+    public void init(FilterConfig arg0) throws ServletException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)